src/plugins/intel_npu/src/plugin/npuw/partitioning/patterns/dcoff.cpp

```diff
@@ -139,15 +135,21 @@ void apply_remap(Subgraph& fcall, const ClosureRemap& m) {
         if (fcall._closure[i]) {
             new_closure.push_back(fcall._closure[i]);
         } else {
-            new_closure.push_back(m.weights_to_unpack.count(i) ? fcall._lazy_closure[i].eval() : fcall._closure[i]);
+            // Only support permute transformations
+            auto transforms = fcall._lazy_closure[i].get_transformations();
+            if (transforms.size() == 2 && std::holds_alternative<ov::npuw::weights::op::Permute>(transforms[0])) {
+                // Assuming we only match weights where DQ did set Transpose after - thus nothing extra to do here
+                new_closure.push_back(ov::Tensor());
```

Comment:
Since it's going to be reused via DCOFF, we should mark those somehow so those will be allocated on CPU

Reply:
But in this case they won't be actually shared in the bank...